spatialos-sdk/src/worker/connection.rs

```diff
@@ -85,25 +82,25 @@ pub trait Connection {
         timeout_millis: Option<u32>,
     ) -> RequestId<EntityQueryRequest>;
 
-    fn send_command_request(
+    fn send_command_request<C: Component>(
         &mut self,
         entity_id: EntityId,
-        request: component::internal::CommandRequest,
+        request: C::CommandRequest,
         timeout_millis: Option<u32>,
-        command_parameters: CommandParameters,
+        allow_short_circuit: bool,
```

Comment:
I would rather not inline this, as I mentioned in the other comment.

Reply:
Agreed with @davedissian - would prefer if the struct was kept